src/status_im/models/browser.cljs

```diff
@@ -82,4 +86,16 @@
      (update :index inc)
 
      (and permission permissions-data)
-     (assoc-in [:permissions-allowed permission] (get permissions-data permission)))))
\ No newline at end of file
+     (assoc-in [:permissions-allowed permission] (get permissions-data permission)))))
+
+(defn web3-send-async [{:keys [db]} {:keys [method] :as payload} message-id]
```

Comment:
same here - param order in model fn - should be e.g. `payload`, `message-id` and then `cofx`